src/main/kubernetes/openshift.yml

```diff
@@ -92,30 +92,40 @@ spec:
             # and 3 pods with low memory perform badly, so we'll have to make do with a single pod.
             - name: discovery.type
               value: "single-node"
-            # Memory locking doesn't work on our OpenShift instance,
-            # but this shouldn't be too bad as we don't expect swapping to be enabled.
-            - name: bootstrap.memory_lock
-              value: "false"
```

Comment:
I'm surprised you don't need that for Elasticsearch? Did it work on OpenShift?

Reply:
yeah, I wasn't sure about this one... I've been going through the env variables we have in Hibernate Search container (for tests) to see which ones we could add here too, and I noticed that this property is only set for the OpenSearch flavor of containers so I just blindly assumed that it's not there for Elasticsearch ... But! looking at the docs: https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-configuration-memory.html#bootstrap-memory_lock it is there, so I'll add it back and probably do the same for Search...